version/version.go

```diff
@@ -0,0 +1,3 @@
+package version
+
+const Version = "v0.0.15"
```

Comment:
Default this to `0.0.0` and we can set it at build time with `go build`. That way we don't have to keep it updated here. We've been using `v0.0.x` for tags and `0.0.x` for version numbers